### PR TITLE
Add AdminCategory Edit, Show, Update

### PR DESCRIPTION
**Pull request by codegracie**
Added remaining Edit, Show and Update functions for Admin Category

### CODE REVIEW

File: app/views/admin/categories/_form.html.erb [hunk 1]
```diff
@@ -2,18 +2,42 @@
...
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          <h2><%= t ".panel_title" %></h2> 
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <%= form_for [:admin, @category], html: {multipart: true,
+              class: "form-horizontal style-form"} do |f| %>
+              <div class="form-group">
+                <%= f.label :name, class: "col-md-2 control-label" %>
+                  <div class="col-md-10">
```
Comment (MuhammadTamzid) [inline app/views/admin/categories/_form.html.erb]: indent

File: app/controllers/admin/categories_controller.rb [hunk 2]
```diff
@@ -29,6 +29,15 @@ def edit
     @category = Category.find params[:id]
   end
 
+  def update
+    @category = Category.find params[:id]
+      if @category.update_attributes category_params
```
Comment (MuhammadTamzid) [inline app/controllers/admin/categories_controller.rb]: indent

File: app/views/admin/categories/_form.html.erb [hunk 3]
```diff
@@ -2,18 +2,42 @@
...
-  <div class="row">
-    <div class="col-md-12">
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          <h2><%= t ".panel_title" %></h2> 
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <%= form_for [:admin, @category], html: {multipart: true,
+              class: "form-horizontal style-form"} do |f| %>
+              <div class="form-group">
```
Comment (MuhammadTamzid) [inline app/views/admin/categories/_form.html.erb]: indent
remove two white space

Comment (codegracie) [inline app/views/admin/categories/_form.html.erb]: i can't seem to find the two white spaces. which you refer? please advise. thanks

File: app/views/admin/categories/_form.html.erb (hunk 1 above)
Comment (codegracie) [inline app/views/admin/categories/_form.html.erb]: modified

File: app/controllers/admin/categories_controller.rb (hunk 2 above)
Comment (codegracie) [inline app/controllers/admin/categories_controller.rb]: modified

File: app/views/admin/categories/_form.html.erb (hunk 3 above)
Comment (MuhammadTamzid) [inline app/views/admin/categories/_form.html.erb]: remove two white space before <div class="form-group">

Comment (codegracie) [inline app/views/admin/categories/_form.html.erb]: hello, i have sought advise from Dieu about this and for him to also help me find the white spaces. 
he replied that he thinks it is ok.

Comment (MuhammadTamzid) [inline app/views/admin/categories/_form.html.erb]: OK

**Pull request merged**